scripts/define_function.sh

```diff
@@ -57,24 +52,22 @@ zds_travis_time_finish() {
 }
 ##
 
-
 ## start fold for travis
 ZDS_SHOW_TRAVIS_FOLD=0
-if $(_in "--travis-output" $@); then
+readonly travis_output="$(_in "--travis-output" "$@")"
```

Comment:
Ce changement n’entraînera aucune erreur du type: "command not found" ?

Reply:
Si, cela peut bien entraîner des erreurs de ce type. J’ai adopté cette forme car elle était équivalente à ce qui est actuellement en production.

Le problème se pose ainsi : veux-on exécuter la commande `_in "--travis-output" "$@"` ou exécuter le résultat de cette commande ? Actuellement, on exécute le résultat. Ça me paraît étrange, mais je ne sais pas ce que l’on veut faire exactement, alors j’ai fait la modification sans changer le fonctionnement. Si on veut seulement exécuter la commande (et pas exécuter la sortie de la commande), il suffit simplement d’enlever `$()`, on obtiendrait alors `if _in "--travis-output" "$@"; then`.

Reply:
On souhaite savoir si "--travis-output" figure en argument